src/pudl/analysis/record_linkage/eia_ferc1_model_config.py

```diff
@@ -33,59 +32,57 @@
     blocking_rule_10,
 ]
 
-plant_name_comparison = ctl.name_comparison(
+plant_name_comparison = cl.NameComparison(
     "plant_name",
-    damerau_levenshtein_thresholds=[],
     jaro_winkler_thresholds=[0.9, 0.8, 0.7],
 )
-utility_name_comparison = ctl.name_comparison(
+utility_name_comparison = cl.NameComparison(
     "utility_name",
-    damerau_levenshtein_thresholds=[],
     jaro_winkler_thresholds=[0.9, 0.8, 0.7],
-    term_frequency_adjustments=True,
-)
-fuel_type_code_pudl_comparison = cl.exact_match(
-    "fuel_type_code_pudl", term_frequency_adjustments=True
 )
+utility_name_comparison.configure(term_frequency_adjustments=True)
+fuel_type_code_pudl_comparison = cl.ExactMatch("fuel_type_code_pudl")
+fuel_type_code_pudl_comparison.configure(term_frequency_adjustments=True)
```

Comment:
This seems like a funny change to me, but the `.configure()` method is used to set attributes which are common to all the different kinds of match classes now.